assignments/assignment_2/index.js

```diff
@@ -0,0 +1,12 @@
+const http=require("http")
+const fs=require("fs")
+fs.writeFileSync("index.html","<h1>Hello world</h1>")
+
+http.createServer(function(req,res){
+    const data=fs.readFileSync("index.html")
+        
+            res.writeHead(200,{"Content-type":"text/html"});
+            res.end();
+        
+
+}).listen(4004)
```

Comment:
listen to 3000, its mentioned in md file, 1 testcase is failing

Reply:
@HarshS15j we can run our server on any port its not at all impactable for test cases and i dont think any test case is failing...
please refer this screenshot 
![image](https://user-images.githubusercontent.com/93022966/147737957-ac9d968b-a47c-400a-8f7c-1194ee4515e5.png)

Reply:
@yogesh-tayo in my case one test case was failing, so better change the port to 3000 as is already mentioned. It's up to you. I didn't take ss so can't show that.